vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -98,28 +84,57 @@ ent jhipster entity Conference
     - `Do you want pagination and sorting on your entity?` (Yes, with infinite scroll)
     - (If you chose to be prompted to generate micro frontends) `Do you want to generate micro frontends?` (Up to you)
 
-
-8. Affirm each overwrite prompt (echoed below) to resolve conflicts as the Blueprint generates controllers, repositories, services and micro frontends for your entity. **Note: Enter "a" in response to the initial prompt to authorize all overwrites to existing files with the necessary configuration changes.**
-    - `Overwrite src/main/resources/config/liquibase/master.xml?` 
+7. Affirm each overwrite prompt to resolve conflicts as the Blueprint generates controllers, repositories, services and micro frontends for your entity. **Note: Enter "a" to the initial prompt to authorize changes to all the updated files.**
 
     - `Overwrite package.json?`
-    - `Overwrite bundle/descriptor.yaml?`
-    - `Overwrite bundle/plugins/jhipster-plugin.yaml?`
-    - `Overwrite src/main/resources/config/liquibase/master.xml?`
-    - `Overwrite src/main/java/com/mycompany/myapp/config/CacheConfiguration.java?`
 
-You have now generated an Entando project, including a Spring Boot microservice with database integration and React-based micro frontends.      
+You have now generated a Spring Boot microservice with database integration and React-based micro frontends.     
 
-### Project Structure
-   * ```/src/main/docker``` contains Docker files to help with local development environments.
+### Output
    * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration.
    * ```/ui``` holds the React-based micro frontends. By default, JHipster generates 3 MFEs per entity to contain the details, form, and table.
-   * ```/bundle``` is used to assemble the project code into an Entando Bundle.
+   * ```/src/main/docker``` contains Docker files to help with local development environments.
+
+Next, move the `/ui` and `/src/main/docker` files into the standard Entando bundle directories and then wire the MFEs to the Spring Boot microservice.
```

Comment:
...where are the corresponding instructions?? not sure what to do here, or why this doesn't translate to numbered steps ??

Reply:
yep, confusing. I don't think this sentence is needed anymore.